res/flow-capture.yml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: netobserv-cli
+  namespace: netobserv-cli
+  labels:
+    app: netobserv-cli
+spec:
+  selector:
+    matchLabels:
+      app: netobserv-cli
+  template:
+    metadata:
+      labels:
+        app: netobserv-cli
+    spec:
+      serviceAccountName: netobserv-cli
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      containers:
+      - name: netobserv-cli
+        # image: quay.io/netobserv/netobserv-ebpf-agent:main
+        image: quay.io/jpinsonn/netobserv-ebpf-agent:flow-stream-1
+        imagePullPolicy: Always
+        securityContext:
+          privileged: true
+          runAsUser: 0
+        env:
+          - name: INTERFACES
+            value: "{{FLOW_FILTER_VALUE}}"
+          - name: SAMPLING
+            value: "1"
+          - name: ENABLE_RTT
+            value: "true"
+          # TODO: find why this breaks collection: EOF
+          #- name: ENABLE_PKT_DROPS
+          #  value: "true"
```

Comment:
Uncomment this to give a try with pktDrop

After some time running it, I get EOF in the TCP connection read